runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -196,6 +193,7 @@
   @VisibleForTesting static final int GCS_UPLOAD_BUFFER_SIZE_BYTES_DEFAULT = 1024 * 1024;
 
   @VisibleForTesting static final String PIPELINE_FILE_NAME = "pipeline.pb";
+  @VisibleForTesting static final String DATAFLOW_GRAPH_FILE_NAME = "dataflow_graph.pb";
```

Comment:
I'm curious how much of the information in `dataflow_graph.pb` is not contained within `pipeline.pb`. As long as Dataflow Service is building the functionality to read the graph from a file, would it be better to consume the `pipeline.pb` directly and work towards deprecating the Dataflow-specific representation at this layer?